<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormulaView</class>
 <widget class="QGroupBox" name="FormulaView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Function</string>
  </property>
  <widget class="QPushButton" name="glnumber">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>383</y>
     <width>58</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Constant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="glvariable">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>383</y>
     <width>58</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="glunary">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>383</y>
     <width>68</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Unary</string>
   </property>
  </widget>
  <widget class="QPushButton" name="glbinary">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>383</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Binary</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Expression Tree</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="additional">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Spectrum axis range and channel width</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Selected element</string>
   </property>
   <widget class="QLineEdit" name="nameedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="minimum">
     <double>-1000000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="applybutton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ParseButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>78</y>
     <width>290</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Try to parse expression from string</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calbutt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>42</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select SP2 for calibration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace with</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>78</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
